sqlmesh/schedulers/airflow/util.py

```diff
@@ -36,13 +38,23 @@
 
 @contextlib.contextmanager
 def scoped_state_sync() -> t.Iterator[StateSync]:
+    state_schema = c.SQLMESH
     try:
         connection = Connection.get_connection_from_secrets(SQLMESH_STATE_CONN_ID)
 
+        connection_config_dict = json.loads(connection.extra)
+        state_schema = connection_config_dict.get("state_schema", state_schema)
+        if "type" not in connection_config_dict:
+            logger.info(
+                "SQLMesh connection in Airflow did not have type defined. "
+                "Therefore using Airflow database connection"
+            )
+            raise AirflowException
```

Comment:
Are we raising a type here? Shouldn't it be `AirflowException()`?

Reply:
I think this works as expected:
```
def main():
    try:
        print("here")
        raise AirflowException
        print("here2")
    except AirflowException:
        print("exception")
```
```
here
exception
```